### ISSUE TITLE
Folder Color Change

### ISSUE THREAD

**Issue by mrcooltools** [Birkbjo/OctoPrint-Themeify#109]
Is it possible to edit or change the color of the created folders only text without changing all the text?

**Comment by Birkbjo** [comment]
Yes. Try something like selector: `#files`, css-rule: `color`, value: `red`.
